docs-react-native/react-native/docs/triggers.md

```diff
@@ -132,17 +132,17 @@ On Android, you have the option to create your trigger notification with Android
 ```js
 const trigger: TimestampTrigger = {
   //...
-  alarmManager: true,
+  alarmManager: true, // uses android's `AlarmManagerCompat.setExact`
 };
 ```
 
-If you want to allow the notification to display when in low-power idle modes, set `allowWhileIdle`:
+If you want to allow the notification to display when in low-power idle modes, use `type` property:
 
 ```js
 const trigger: TimestampTrigger = {
   //...
   alarmManager: {
-    allowWhileIdle: true,
+    type: AlarmType.SET_EXACT_AND_ALLOW_WHILE_IDLE,
```

Comment:
from `https://developer.android.com/develop/background-work/services/alarms/schedule#repeating`:
```
A poorly-designed alarm can cause battery drain and put a significant load on servers. For this reason, on Android 4.4 (API level 19) and higher, all repeating alarms are [inexact alarms](https://developer.android.com/develop/background-work/services/alarms/schedule#inexact).
```
Would be nice if this is documented somewhere